src/App.svelte

```diff
@@ -0,0 +1,236 @@
+<script>
+
+	/* ### VARIABLES ### */
+
+		import Navigation from './components/Navigation.svelte';
+		import Results from './components/Results.svelte';
+		import AuditForm from './components/AuditForm.svelte';
+		import Options from './components/Options.svelte';
+		import About from './components/About.svelte';
+		
+		const versions = ['beta']; // Available versions (stored in /public/rgesn)
+		let defaultVersion = 'beta';
+		let referential; // RGESN content
+		let index = 0; // Current audit index identifier
+		let audits = [ // Audits progression
+			{  
+				byCriteria: {},
+				byCounters: {
+					satisfied: 0,
+					rejected: 0,
+					notApplicable: 0
+				},
+				selectedVersion: defaultVersion, // By default.
+			}
+		];
+		let render = false; // Makes Results & AuditForm components reactives
+		const env = window.browser === undefined ? window.chrome : window.browser; // Makes extension cross platform
+
+	/* ### FUNCTIONS ### */
+
+		function getRGESN(versionToLoad) {
+			return new Promise((resolve, reject) => {
+				fetch(`./rgesn/${versionToLoad}.json`)
```

Comment:
@AwebsomeFr où est le répertoire rgesn ?

Reply:
La correspondance se fait avec le dossier : `/public/rgesn/`.

Reply:
`const versions = ['beta']; // Available versions (stored in /public/rgesn)`

Reply:
merci !